Examples/Scripts/TrackingPerformance/reconstructionPerformance.C

```diff
@@ -0,0 +1,166 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2019-2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <array>
+#include <string>
+#include <vector>
+#include <iostream>
+
+#include <TCanvas.h>
+#include <TEfficiency.h>
+#include <TProfile.h>
+#include <TFile.h>
+
+#include "CommonUtils.h"
+
+void reconstructionPerformance(std::vector<std::string> inputFileNames) {
+  std::array<TCanvas*, 3> emho = {nullptr, nullptr, nullptr};
+  std::vector<std::string> tags = {"eta", "pT"};
+  std::vector<std::string> params = {"trackeff", "nMeasurements", "nHoles",
+                                    "nOutliers", "nStates"};
+
+  // Create the Canvae
+  unsigned int itag = 0;
+  for (const auto& t : tags) {
+    std::string name = "efd_" + t;
+    std::string label = "Efficiency, Fakes, Duplicate vs. " + t;
+    emho[itag] =
+        new TCanvas(name.c_str(), label.c_str(), itag * 10, 10, 1800, 300);
+    emho[itag]->Divide(5, 1);
+    ++itag;
+  }
+
+  unsigned int imode = 1;
+  for (auto fileName : inputFileNames) {
+    // Te approprate file
+    auto file = TFile::Open(fileName.c_str(), "read");
```

Comment:
Shall we throw a failure if `file == nullptr`?

Reply:
I will do this in another iteration.